data/preset/blog.yaml

```diff
@@ -0,0 +1,181 @@
+          slots:
+            contents:
+            - shortdef: 'Banner.Set(halign: start)'
+              slots:
+                card: 'Card.Title(halign: center)'
+            - type: ContentGroup.ContentGroup
+              properties:
+                halign: fill
+                expand: true
+              slots:
+                arrangement:
+                  type: Arrangement.List
+                  slots:
+                    card: Card.DefaultFamily
+                selection:
+                  type: Selection.ArticlesForSet
+                  id: set-articles
+                  slots:
+                    order: 'Order.PublishedDate(ascending: false)'
+
+- &search-page
+  shortdef: 'Layout.InfiniteScrolling(expand: true)'
+  references:
+    lazy-load: search-results
+  slots:
+    content:
+      type: Layout.ResponsiveMargins
+      slots:
+        content:
+          shortdef: 'Layout.Box(orientation: vertical)'
+          slots:
+            contents:
+            - shortdef: 'Banner.Search(halign: start)'
+              slots:
+                card: 'Card.Title(halign: center)'
+            - type: ContentGroup.ContentGroup
+              properties:
+                halign: center
+              slots:
+                arrangement:
+                  type: Arrangement.List
+                  slots:
+                    card: Card.List
+                selection:
+                  type: Selection.Search
+                  id: search-results
+                no-results:
+                  shortdef: 'Layout.Box(orientation: vertical)'
+                  slots:
+                    contents:
+                    - type: ContentGroup.NoResultsMessage
+                      properties:
+                        halign: start
+                        justify: left
+                    - type: ContentGroup.ContentGroup
+                      styles:
+                        - search-suggestions
+                      properties:
+                        valign: end
+                      slots:
+                        title:
+                          type: ContentGroup.StaticText
+                          properties:
+                            label: !translate 'Latest posts'
+                            halign: start
+                        arrangement:
+                          shortdef: 'Arrangement.SquareGuys(max-rows: 1)'
+                          slots:
+                            card:
+                              type: Card.DefaultFamily
+                              properties:
+                                excluded_types:
+                                - 0
+                                - 2
+                        selection:
+                          type: Selection.AllHighlights
+                          slots:
+                            filter: 'Filter.Sets(invert: true)'
+                            order: 'Order.PublishedDate(ascending: false)'
+
+root:
+  type: Controller.Buffet
```

Comment:
Consider `Controller.Mesh`? As far as I can tell from looking at the code, the only thing `Controller.Buffet` has that `Controller.Mesh` doesn't, is support for a context bar, which we don't use.